test/install_gcloud_test.go

```diff
@@ -0,0 +1,20 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/shell"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInstallGCloud(t *testing.T) {
+	t.Parallel()
+
+	require.NoError(t, runInstallGCloudScript(t))
+}
+
+func runInstallGCloudScript(t *testing.T) error {
+	cmd := shell.Command{Command: "../modules/gcp-helpers/bin/install-gcloud"}
```

Comment:
Will calling this script in a test that runs in a build job running in CircleCi where we _already_ ran this script cause any issues?

Reply:
Hmmmm, well whilst it's non-ideal I guess all of the commands in the script are idempotent except adding the `apt` sources. The only way around I guess would be to run this specific test in its own isolated Docker executor which is also a bit of an overkill. Thoughts?

Reply:
No strong opinion. You could run the entire test in a Docker container, or a separate CircleCi executor, or leave as-is. Your call!

Reply:
Well I kind of need `gcloud` installed for other tests in a separate (and future) branch(es), so I'll leave it as-is, but nice pickup!